src/content/React-CSharp/ClientApp/src/custom.css

```diff
@@ -3,6 +3,10 @@ a {
   color: #0366d6;
 }
 
+.btn:focus, .btn:active:focus {
```

Comment:
Can we make this change for box shadow focus to all the templates?  Not just react/angular?  Basically this is indeed the same issue I was trying to fix which didn't work because the identity site css isn't appied

Reply:
We probably can, but the other templates are in a different repo so that goal wouldn't affect this PR.